runtimes/common/src/constants.rs

```diff
@@ -194,11 +205,11 @@ pub mod governance {
 }
 
 pub mod did {
-	use crate::BlockNumber;
+	use super::*;
 
-	use super::{Balance, HOURS, KILT, MILLI_KILT};
+	pub const MAX_DID_BYTE_LENGTH: u32 = 7418;
```

Comment:
C: Add comments for these "magic numbers" explaining that these numbers were actually derived in the runtime unit tests.

Reply:
[bb39a1b](https://github.com/KILTprotocol/mashnet-node/pull/329/commits/bb39a1b29f6489dba3b51f5b95c6c9efdd7fc5b7)